.editorconfig

```diff
@@ -0,0 +1,19 @@
+# To use this config with your editor, follow the instructions at:
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+insert_final_newline = true
+
+[*.{c,h}]
+indent_style = space
+indent_size = 4
+
+[{Makefile.*,*.mk}]
+indent_style = tab
```

Comment:
not sure the usage of this config
it could be used to unify the code auto align style? no matter they are using vim or other editor?

Reply:
Nearly every editor out there support this (either built-in or via a simple plugins) - vim, emacs, eclipse, visual studio code, etc see http://editorconfig.org

Since it's used by the editor it will leave "less work" for the `style_check` script. The two are effectively complementing each other - they are not exclusive.

Reply:
I quick checked the android.mk file, it just one space https://github.com/intel/libva/blob/master/Android.mk#L4, so, how about this take effect?

Reply:
Seems like a spurious white-space that can/should be removed. It bares no change in functional behaviour.

Reply:
@XinfengZhang humble ping. Is there anything I can do to move this forward?